plugin/trino-datasketches/src/main/java/io/trino/plugin/datasketches/theta/SketchFunctionsPlugin.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.datasketches.theta;
+
+import com.google.common.collect.ImmutableSet;
+import io.trino.spi.Plugin;
+
+import java.util.Set;
+
+public class SketchFunctionsPlugin
+        implements Plugin
+{
+    @Override
+    public Set<Class<?>> getFunctions()
+    {
+        return ImmutableSet.<Class<?>>builder()
+                .add(Estimate.class)
+                .add(Union.class)
+                .add(UnionWithParams.class)
+                .build();
```

Comment:
This pollutes the global function namespace -- it's the old way for plugins to expose functions. The functions should be exposed via dynamic function resolution so that they can be namespaced properly.